plume-models/src/blogs.rs

```diff
@@ -66,32 +66,32 @@ impl Blog {
     find_by!(blogs, find_by_ap_url, ap_url as String);
     find_by!(blogs, find_by_name, actor_id as String, instance_id as i32);
 
-    pub fn get_instance(&self, conn: &PgConnection) -> Instance {
+    pub fn get_instance(&self, conn: &Connection) -> Instance {
         Instance::get(conn, self.instance_id).expect("Couldn't find instance")
     }
 
-    pub fn list_authors(&self, conn: &PgConnection) -> Vec<User> {
+    pub fn list_authors(&self, conn: &Connection) -> Vec<User> {
         use schema::blog_authors;
         use schema::users;
         let authors_ids = blog_authors::table.filter(blog_authors::blog_id.eq(self.id)).select(blog_authors::author_id);
-        users::table.filter(users::id.eq(any(authors_ids)))
+        users::table.filter(users::id.eq_any(authors_ids))
```

Comment:
from the diesel docs:
> On PostgreSQL, you should use eq(any()) instead. This method may change in the future to automatically perform = ANY on PostgreSQL


have you noticed any performance degradation so far?

Reply:
No, and actually, for now `eq_any()` is equivalent to `eq(any())` with Postgres, but also supports SQlite.